docs/r2/tensorboard_profiling_keras.ipynb

```diff
@@ -998,9 +998,9 @@
       "cell_type": "markdown",
       "source": [
         "## Other ways for profiling\n",
-        "In addition to TensorBoard callback, TensorFlow also provides two additional way to trigger profiler manually: **Profiler APIs** and **Profiler Service**.\n",
+        "In addition to TensorBoard callback, TensorFlow also provides two additional way to trigger profiler manually: *Profiler APIs* and *Profiler Service*.\n",
         "\n",
-        "**NOTE**: Please don't run multiple profilers at the same time. If you want to use either Profiler APIs or Profiler Service with TensorBoard callback, ensure the **profile_batch** parameter is set to 0.\n",
+        "**Note:** Please don't run multiple profilers at the same time. If you want to use either Profiler APIs or Profiler Service with TensorBoard callback, ensure the `profile_batch` parameter is set to 0.\n",
```

Comment:
This one should just be `Note:` (it's a styling element used for devsite callout boxes)

Reply:
Ah, got it. Thanks!